### ISSUE TITLE
WPF ImageView Excepts on load if not given an image

### ISSUE THREAD

**Issue by rooly4** [picoe/Eto#327]
Title says it all, an ImageView object without an Image will throw an exception in the Application.Run context with the phrase
```
System.ArgumentException: Width and Height must be non-negative.
```

**Comment by rooly4** [comment]
Workaround is to fill with placeholder image data before OnLoad is called

**Comment by Panzerfather** [comment]
Which version of Eto do you use? On the devel-branch I can create an ImageView-Control without setting an Image and don't get an error.

**Comment by rooly4** [comment]
nuget package 2.0.7

**Comment by Panzerfather** [comment]
So I think this problem is fixed with version 2.1. When it's out please recheck it or you could compile your own sources from the devel branch right now.

**Comment by cwensley** [comment]
Yeah, I can verify running using the 2.1 develop nuget packages seems to work.

Thanks @Panzerfather for checking that out!  It's really appreciated.
